packages/sdk/src/oracle/tests/optimisticV2.e2e.ts

```diff
@@ -0,0 +1,94 @@
+import dotenv from "dotenv";
+import assert from "assert";
+
+import { Client } from "../client";
+import factory from "../optimisticV2Factory";
+import { getFlags, getAddress } from "../utils";
+import Store from "../store";
+import * as types from "../types";
+
+dotenv.config();
+assert(process.env.PROVIDER_URL_137, "requires PROVIDER_URL_137");
+
+const optimisticOracleAddress = getAddress("0xee3afe347d5c74317041e2618c49534daf887c24");
```

Comment:
I think this might be the wrong address, looks like it's the Skinny Optimistic Oracle.

Reply:
i think we are good https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json#L32

Reply:
Actually, that's the address listed for the OOv2 on Polygon but it's a duplicate of the Skinny OO address on mainnet.

https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/1.json#L141
https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json#L32

Bizarre.